terraform/modules/csb/broker_stig/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.15"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "< 6.0.0"
+    }
+    cloudfoundry = {
+      source  = "cloudfoundry-community/cloudfoundry"
+      version = "< 1.0"
+    }
```

Comment:
I don't think we're using this provider anywhere in the code?

Reply:
And if you were, you'd want to use `cloudfoundry/cloudfoundry`, not the older community version